### PR TITLE
Add Kamiko

### PR DESCRIPTION
**Pull request by carpevenenum**
### kamiko is teh best!!! ive disallowed edits from maintainers because she already teh bestest why would you change her!!!

### CODE REVIEW

File: Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+LikesOccult|-54|
+生|True|
+クーデレ|False|
+忠実|True|
+声|1|
+Lingo Great|sick-ass|
+Lingo Crappy|bad|
+Class1_Proficiency|0|
+Class2_Proficiency|0|
+Class3_Proficiency|255|
```
Comment (Arbiter1223) [inline Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student]: All class proficiencies need to be set to 0 by default. (You shouldn't have preexisting grades for a class if you're just starting the school year, right?)

File: Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Class4_Proficiency|0|
+Class5_Proficiency|0|
+Class6_Proficiency|0|
+Class7_Proficiency|0|
+Class8_Proficiency|0|
+Accent|0|
+Energy|20|
```
Comment (Arbiter1223) [inline Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student]: Energy needs to be set to 2000, so students start with full energy. At this setting, she'd pass out before even making it to school! D:

File: Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ThighHigh Base|23|
+ThighHigh Extra|0|
+骨盤&太もも|2|
+声ピッチ|3|
+頭のアクセサリーの色|1|
+おっぱい|2|
+Lingo Very|14|
+(╯°□°）╯You ruined it!! You killed the magic!!
```
Comment (Arbiter1223) [inline Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student]: Everything else looks okay. Just fix those two things, and she'll be good to go. :D

File: Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student (hunk 2 above)
Comment (carpevenenum) [reply Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student]: fixed!!

File: Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student (hunk 1 above)
Comment (carpevenenum) [reply Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student]: fixed!!

**Comment by carpevenenum** [comment]
i've fixed everything now!!

**Review by Arbiter1223 — APPROVED** [review]
Alright, looks good to go!

**Pull request merged**